### ISSUE TITLE
Download Images

### ISSUE THREAD

**Issue by tichnas** [gsc2001/IIIT-GPS#88]
We should download at least 2-3 more images to put on course cards.

**Comment by gsc2001** [comment]
You do it :stuck_out_tongue:

**Comment by tichnas** [comment]
@gsc2001 You know my choice. Btw how to put emoji?

**Comment by gsc2001** [comment]
Icons dekh le

**Comment by gsc2001** [comment]
You mean here?

**Comment by tichnas** [comment]
@gsc2001 Yes in between the comment. Like the one in your first comment here.

**Comment by gsc2001** [comment]
You have to type so text inside :: like for this :stuck_out_tongue: you type stuck_out_tongue

**Comment by tichnas** [comment]
omg. Nice :+1:
